src/prog_models/utils/calc_error.py

```diff
@@ -170,18 +150,11 @@ def MSE(m, times: List[float], inputs: List[dict], outputs: List[dict], **kwargs
     if not isinstance(outputs[0], m.OutputContainer):
         outputs = [m.OutputContainer(z_i) for z_i in outputs]
 
-    # Checks stability_tol is within bounds
-    # Throwing a default after the warning.
-    if stability_tol >= 1 or stability_tol < 0:
-        warn(f"configurable cutoff must be some float value in the domain (0, 1]. "  
-             f"Received {stability_tol}. Resetting value to 0.95")
-        stability_tol = 0.95
```

Comment:
I don't think we can delete this line. In the main calc_error function in prognostics_model, you check if stability_tol is in the bounds and re-define it as 0.95 if not. However, then you pass kwargs, which still has the original stability_tol value, into the actual error method calculation. So without this check here, stability_tol could still be outside of the bounds. 

I think we either need to include this (in all methods), or maybe re-define kwargs in calc_error after the check that it is within bounds.

Reply:
Ah, great catch. I did not see this happening.

Rather than keep these checks, how about I simply change the kwargs['stability_tol'] instead of the other stability_tol that is created? This way we can keep passing in kwargs and we can keep this section of code deleted.

Reply:
I like this option! Nice fix